lib/mathable.rb

```diff
@@ -0,0 +1,25 @@
+module Mathable
+
+  def average(num1, num2)
```

Comment:
I really like having this in a module. I can definitely use it in my statistics.